doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,87 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
```

Comment:
Is the plan to have `qml.jordan_wigner` at top level? I couldn't find it in the built docs.